### PR TITLE
Leftovers of PipeineOperations -> PipelineBackend renaming

### PR DESCRIPTION
**Pull request by dvadym**
This PR mostly renames ops -> backend, which wasn't renamed on [PR](https://github.com/OpenMined/PipelineDP/pull/148)

### CODE REVIEW

File: pipeline_dp/dp_engine.py [hunk 1]
```diff
@@ -227,38 +227,39 @@ def sample_unique_elements_fn(pid_and_pks):
...
+        col = self._backend.keys(col,
+                                 "Drop accumulators, keep only partition keys")
 
         return col
 
     def _drop_not_public_partitions(self, col, public_partitions,
                                     data_extractors: DataExtractors):
-        """Drops partitions in `col` which are not in `public_partitions`."""
-        col = self._ops.map(
+        """Drbackend partitions in `col` which are not in `public_partitions`."""
```
Comment (miracvbasaran) [inline pipeline_dp/dp_engine.py:254]: "Drops". Find-and-replace can be tricky sometimes. :)

File: tests/pipeline_backend_test.py [hunk 2]
```diff
@@ -34,8 +34,8 @@ def test_filter_by_key_must_not_be_none(self):
             col = p | "Create PCollection" >> beam.Create(data)
             key_to_keep = None
             with self.assertRaises(TypeError):
-                result = self.ops.filter_by_key(col, key_to_keep,
-                                                "filte_by_key")
+                result = self.backend.filter_by_key(col, key_to_keep,
+                                                    "filte_by_key")
```
Comment (miracvbasaran) [inline tests/pipeline_backend_test.py:38]: nit: "filter_by_key". Also in the rest of this file.

File: tests/pipeline_backend_test.py [hunk 3]
```diff
@@ -116,109 +118,112 @@ def _create_mock_pcollection():
...
                                         expected_regex="A transform with label "
-                                        "\"SAME_MAP_NAME_SAME_OPS_SUFFIX\" "
+                                        "\"SAME_MAP_NAME_SAME_backend_SUFFIX\" "
                                         "already exists in the pipeline"):
-                ops_2.map(col, lambda x: x, "SAME_MAP_NAME")
+                backend_2.map(col, lambda x: x, "SAME_MAP_NAME")
 
     def test_one_suffix_multiple_same_stage_name(self):
-        ops = BeamBackend("UNIQUE_OPS_SUFFIX")
+        backend = BeamBackend("UNIQUE_backend_SUFFIX")
```
Comment (miracvbasaran) [inline tests/pipeline_backend_test.py:211]: Should this be in capital letters? (Also in the rest of this file)

**Review by miracvbasaran — CHANGES_REQUESTED** [review]
Looks good overall. Have you looked into if there are any other usages of "ops" in the project that needs to be replaced?

File: pipeline_dp/dp_engine.py (hunk 1 above)
Comment (dvadym) [reply pipeline_dp/dp_engine.py]: Thanks! I should have used whole words replacement :)

File: tests/pipeline_backend_test.py (hunk 2 above)
Comment (dvadym) [reply tests/pipeline_backend_test.py]: Thanks!

File: tests/pipeline_backend_test.py (hunk 3 above)
Comment (dvadym) [reply tests/pipeline_backend_test.py]: Thanks!

**Review by dvadym — COMMENTED** [review]
Thanks for review!

**Comment by dvadym** [comment]
> Looks good overall. Have you looked into if there are any other usages of "ops" in the project that needs to be replaced?

I've checked. ops is not used anymore in the project

**Pull request merged**